ffmpegdecklink.rb

```diff
@@ -150,9 +150,19 @@ def install
     if build.with? "tools"
       system "make", "alltools"
       bin.install Dir["tools/*"].select { |f| File.executable? f }
+      mv bin/"python", pkgshare/"python", :force => true
```

Comment:
Not blocking, but just out of curiosity, what necessitated this addition?

Reply:
Actually just to make `brew audit --strict` happy, because otherwise a `python` folder inside the `bin` folder would be created. That’s the solution agreed by the “normal”  FFmpeg formula. (Here we could also delete the installation of `tools`, because we don’t need it.)

Reply:
ah cool. Thanks!